src/explore-education-statistics-frontend/src/hooks/useCookies.ts

```diff
@@ -0,0 +1,57 @@
+import { useCookies as useBaseCookies } from 'react-cookie';
+import {
+  enableGA,
+  disableGA,
+  googleAnalyticsCookies,
+} from '@frontend/services/googleAnalyticsService';
+import { addMonths } from 'date-fns';
+
+interface Cookie {
+  name: string;
+  expires: Date;
+  duration?: string;
+}
+
+interface CookieMap {
+  bannerSeen: Cookie;
+  disableGA: Cookie;
+}
+
+export const cookieMap: CookieMap = {
+  bannerSeen: {
+    name: 'ees_banner_seen',
+    expires: addMonths(new Date(), 1),
+    duration: '1 month',
+  },
+  disableGA: {
+    name: 'ees_disable_google_analytics',
+    expires: addMonths(new Date(), 1),
+    duration: '1 month',
+  },
+};
+
+export function useCookies() {
+  const [liveCookies, setCookie, removeCookie] = useBaseCookies();
+
+  return {
+    getCookie(cookieKey: keyof CookieMap) {
+      return liveCookies[cookieMap[cookieKey].name];
+    },
+    setBannerSeenCookie(isSeen: boolean) {
+      setCookie(cookieMap.bannerSeen.name, isSeen, {
+        expires: cookieMap.bannerSeen.expires,
+      });
+    },
+    setGACookie(isDisabled: boolean) {
```

Comment:
Would probably rename this to `setGADisabledCookie` to be consistent with the actual name

Reply:
[Analytics.js used a falsy property](https://developers.google.com/analytics/devguides/collection/analyticsjs/user-opt-out) to disable GA, and so I followed suit so that the Cookie and the window property that are disabling the analytics have the same value.

I'll rename to `setGADisabledCookie`